package.json

```diff
@@ -65,15 +65,13 @@
       ]
```

Comment:
Shouldn't this file have `"version": "1.0.0"` like the other package.jsons have?

Reply:
Oh oops, this is the monorepo package.json, nevermind.